docs/examples_benchmark.md

```diff
@@ -0,0 +1,68 @@
+# Real-world Example Benchmarks
+For all three real-word examples (PNEUMONIA, NER, CCFRAUD), we perform benchmark testing to assess three main aspects: the training overhead, model performance, and scalability of training under FL setting. The main purpose of this benchmark is to show that FL has been implemented correctly and works as expected, rather than to focus on the exact metric value shown below, which will vary based on distinct models and datasets. In the following experiments, each silo is trained with the same hyperparameters to ensure the consistency of results, and the model performance was assessed with all test data combined from silos. Below are the results for each aspect.
```

Comment:
Here, I would use this introduction to provide the answers as well as the questions, to avoid suspense.

Example:

For all three real-word examples (PNEUMONIA, NER, CCFRAUD), we performed a benchmark to assess three main aspects: the training overhead, model performance, and scalability.

This investigation was based on the real-world examples provided in this repository. It shows that the FL on AzureML implementation is:
- efficient: overhead is kept minimal (see section 1),
- valid: reaching parity in quality (see section 2),
- scalable: it can adapt to larger datasets by leveraging distributed training (see section 3).